### PR TITLE
Changed build requirements packages for Centos

### PR DESCRIPTION
**Pull request by danpawlik**
Some packages have been changed the name, so
building bup was problematic.

We discuss code changes on the mailing list bup-list@googlegroups.com,
but if you'd prefer to begin the process with a pull request, that's
just fine.  We're happy to have the help.

In any case, please make sure each commit includes a

  Signed-off-by: Someone <someone@some.where>

line in the commit message that matches the "Author" line so that
we'll be able to include your work in the project.  See
./SIGNED-OFF-BY for the meaning:

  https://github.com/bup/bup/blob/master/SIGNED-OFF-BY

After you submit the pull request, someone will eventually redirect it
to the list for review, and you will of course be included in the
conversation there.

On the other hand, if you're comfortable with "git send-email" (or the
equivalent), please post your patches to the list as described in the
"Submitting Patches" section in ./HACKING:

  https://github.com/bup/bup/blob/master/HACKING

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,11 +161,11 @@ From source
 
     ```sh
     yum groupinstall "Development Tools"
-    yum install python python-devel libacl-devel
+    yum install python2 python2-devel libacl-devel pylibacl
     yum install fuse-python pyxattr
     yum install perl-Time-HiRes
     yum install readline-devel # optional (bup ftp)
-    yum install python-tornado # optional (bup web)
+    yum install python-tornado-doc # optional (bup web)
```
Comment (jmberg) [inline README.md:168]: that seems odd, why would we care about the documentation?

**Comment by rlbdv** [comment]
...and once we've consider jmberg's question, would you be comfortable with a Signed-off-by psuedo-header in the commit?

https://github.com/bup/bup/blob/master/SIGNED-OFF-BY

Thanks

File: README.md (hunk 1 above)
Comment (danpawlik) [reply README.md:168]: If we don't care, I can change it like it was earlier.

**Comment by danpawlik** [comment]
@rlbdv  Sure.

File: README.md (hunk 1 above)
Comment (danpawlik) [reply README.md]: @jmberg  done

**Comment by danpawlik** [comment]
Do we care about Centos 8? Current package build is focused on Centos 7.

**Comment by rlbdv** [comment]
Without knowing Centos well, I'd imagine ideally, we'd probably want to focus (at least) on whatever the current stable release is.

**Comment by rlbdv** [comment]
Incorporated as ec26ee2a6b6427e929b7fee2318320c495a2a0aa.

**Pull request closed without merging**